src/python/utils/stopword_model.py

```diff
@@ -0,0 +1,33 @@
+from sets import Set
+from datetime import datetime
+
+
+class StopWordModel(object):
```

Comment:
Can you add a verify() method that just does a set comparison with the valid fields? The verify method can be called by operations modifying the database so we can ensure that all models submitted to the DB are valid.